### PR TITLE
fixed slider drag on handle disabled

### PR DESCRIPTION
**Pull request by alimemonzx**
Fixed the bug #938, there are no test included for this small fix. Please let me know if any other changes required.

### CODE REVIEW

File: src/nouislider.core.less [hunk 1]
```diff
@@ -9,7 +9,7 @@
 -webkit-tap-highlight-color: rgba(0,0,0,0);
 -webkit-user-select: none;
 -ms-touch-action: none;
-	touch-action: none;
+	touch-action: pan-y;
```
Comment (leongersen) [inline src/nouislider.core.less]: Why are you changing touch action to `pan-y` here?

**Comment by leongersen** [comment]
I've merged part of this in the branch for milestone 13.0.0.

**Comment by leongersen** [comment]
#938 is fixed in noUiSlider 13.0.0. Thanks for [becoming a contributor!](https://github.com/leongersen/noUiSlider/graphs/contributors)

I wasn't able to directly merge this PR due to the `touch-action` change, but I've merged commit da3b16.

File: src/nouislider.core.less (hunk 1 above)
Comment (alimemonzx) [reply src/nouislider.core.less]: Any update on this?

Comment (leongersen) [reply src/nouislider.core.less]: Please discuss this in #788

**Pull request closed without merging**